plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -188,11 +188,10 @@ func buildControllerRoles() ([]rbacv1.ClusterRole, []rbacv1.ClusterRoleBinding)
 			rbacv1helpers.NewRule("get", "list", "watch", "update", "patch").Groups(legacyGroup).Resources("persistentvolumes").RuleOrDie(),
 			rbacv1helpers.NewRule("update", "patch").Groups(legacyGroup).Resources("persistentvolumeclaims/status").RuleOrDie(),
 			rbacv1helpers.NewRule("get", "list", "watch").Groups(legacyGroup).Resources("persistentvolumeclaims").RuleOrDie(),
-			// glusterfs
```

Comment:
is there an out of tree CSI gluster controller now? does it deploy its own role/rolebindings to grant required permissions or reference this default role? do we need to open an issue to update the manifest there to add these removed permissions?

Reply:
I am not sure if there is a supported glusterfs csi driver currently. The driver Google shows is deprecated - https://github.com/gluster/gluster-csi-driver 

But it yes, the permissions associated with expand-controller which was given here for intree glusterfs driver shouldn't matter, any external CSI driver should run with its own serviceaccount and permissions.

Reply:
>(does it need storageclass read permission independent of glusterfs?)

It shouldn't. The only other place we do read storageclasses for expansion purposes (apart from external-resizer which runs with its own permissions), is for admission -  https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/storage/persistentvolume/resize/admission.go#L62

But that code shouldn't run with permissions specified in here. So, we should be fine. I triggered alpha test suite just case, because it has more of expansion related tests to be sure.